sky/shell/ui/engine.h

```diff
@@ -94,6 +96,8 @@ class Engine : public UIDelegate,
   gfx::Size physical_size_;
   std::string initial_route_;
   blink::SkyDisplayMetrics display_metrics_;
+  std::string language_code_;
+  std::string country_code_;
```

Comment:
I thought that part of the rationale for the caching in Engine was that a call to OnViewportMetricsChanged or OnLocaleChanged could arrive before SkyView is instantiated.

Actually, that's happening now because in SkyActivity I'm calling onLocaleChanged before runFromBundle.